ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -3632,4 +3632,13 @@ DRSM8X:
   - dryer_run_status_7
   - dryer_run_status_8
   implements:
-  - MONITORING
\ No newline at end of file
+  - MONITORING
+
+DICM:
+  id: "4953043284228747776"
+  description: "Damper isolation control and monitoring."
+  is_abstract: true
+  uses:
+   - supply_air_isolation_damper_closed_status
+   - supply_air_isolation_damper_open_status
```

Comment:
Same as above.

Reply:
they have been added in the telemetry fields.

Reply:
yes, after another round of internal team meeting.